org.jacoco.doc/docroot/doc/faq.html

```diff
@@ -45,13 +45,13 @@
   duplicate classes or create separate reports or report groups for each version.
 </p>
 
-<h3>Code with exceptions shows no coverage. Why?</h3>
+<h3>Source code lines with exceptions shows no coverage. Why?</h3>
 <p>
   JaCoCo determines code execution with so called probes. Probes are inserted
   into the control flow at certain positions. Code is considered as executed
   when a subsequent probe has been executed. In case of exceptions such a
-  sequence of instructions is aborted somewhere in the middle and not marked as
-  executed.
+  sequence of instructions is aborted somewhere in the middle and the
+  corresponding source code lines is not marked as executed.
```

Comment:
I feel that this should be refined - after the change I still get impression that exceptions are completely not covered by JaCoCo.